client/src/components/shopModal.jsx

```diff
@@ -2,15 +2,135 @@ import React from 'react';
 
 const ShopModalBase = window.styled.div`
 position: absolute;
+font-family: Nunito Sans;
+top: 50px;
+width: 100vw;
 background-color: black;
 color: white;
-width: 100%;
 transition: opacity .5s ease;
 opacity: ${(props) => props.opacity};
+display: flex;
+justify-content: center;
+`;
+const Row = window.styled.div`
+padding: 2em
+display: flex;
+flex-wrap: wrap;
+`;
+const Column = window.styled.div`
+display: flex;
+flex-wrap: nowrap;
+flex-direction: column;
+padding: 0 .5em;
+`;
+const RedColumn = window.styled.div`
+display: flex;
+color: red;
+flex-wrap: nowrap;
+flex-direction: column;
+padding: 0 .5em;
 `;
 const ShopModal = ({ opacity }) => {
   return (
-    <ShopModalBase opacity={opacity}>Hello World</ShopModalBase>
+    <ShopModalBase opacity={opacity}>
+      <Row>
+
+      <Column>
+        <h4>Women's</h4>
+        <a>Shop All</a>
+        <a>New Arrivals</a>
+        <a>Jackets &amp; Vests</a>
+        <a>Fleeces</a>
+        <a>Sweatshirts &amp; Hoodies</a>
+        <a>Sweaters</a>
+        <a>Shirts</a>
+        <a>T-Shirts</a>
+        <a>Pants &amp; Jeans</a>
+        <a>Snow &amp; Alpine Pants</a>
+        <a>Shorts</a>
+        <a>Overalls &amp; Jumpsuits</a>
+        <a>Swimwear</a>
+        <a>Wetsuits</a>
+        <a>Waders &amp; Fishing Gear</a>
+        <a>Base Layers</a>
+        <a>Underwear &amp; Sports Bras</a>
+        <a>Socks</a>
+        <a>Hats &amp; Accessories</a>
+        <a>Work Wear</a>
+        <a>Used Gear</a>
+      </Column>
+      <Column>
+        <h4>Men's</h4>
+        <a>Shop All</a>
+        <a>New Arrivals</a>
+        <a>Jackets &amp; Vests</a>
+        <a>Fleeces</a>
+        <a>Sweatshirts &amp; Hoodies</a>
+        <a>Sweaters</a>
+        <a>Shirts</a>
+        <a>T-Shirts</a>
+        <a>Pants &amp; Jeans</a>
+        <a>Snow &amp; Alpine Pants</a>
+        <a>Shorts</a>
+        <a>Boardshorts</a>
+        <a>Wetsuits</a>
+        <a>Waders &amp; Fishing Gear</a>
+        <a>Base Layers</a>
+        <a>Underwear</a>
+        <a>Socks</a>
+        <a>Hats &amp; Accessories</a>
```

Comment:
should we all get matching Patagonia hats and wear them during our presentation?